### ISSUE TITLE
Make 08-sharing-images.md work through VS Code container

### ISSUE THREAD

**Issue by MichaelMadsen** [praqma-training/docker-katas#110]
If the exercises are being run in a `praqma/vscode` container, it is not currently possible to `docker login`, since it saves the credentials within the container, and the docker commands are being run from the host.

One way of solving this is to install a different authentication method inside `praqma/vscode`:
`apt install gnupg2 pass`
(https://stackoverflow.com/questions/50151833/cannot-login-to-docker-account)

We could add it to the exercise as a hint:
>Hint: If authentication fails, you can run `apt install gnupg2 pass` and try again

Maybe a better solution is to update `praqma/vscode` to include these packages. However, I have no idea where this docker image actually comes from. For now I am documenting the issue here. Will explore further...

**Comment by MichaelDepner** [comment]
On further inspection, looks like the problem goes beyond volume mounts:
```Error saving credentials: error storing credentials = err: exit status 1, out: `cannot autolaunch D-Bus without X11 $DISPLAY` ```
From [this issue](https://github.com/docker/compose/issues/6023), it looks like the docker-compose installation also installs `golang-docker-credential-helpers`, which docker attempts to use as default credentials helper.

Looking at [the bug report](https://bugs.launchpad.net/ubuntu/+source/docker-compose/+bug/1796119), it seems the workaround is either to install `pass` as described above, or to `sudo rm /usr/bin/docker-credential-secretservice`. We cannot uninstall it, since `docker-compose` has a hard dependency to it for some reason...

I gave it a try with out infrastructure, and docker login works after running the workaround.

TL:DR - here are the immediate things we can do:
- Add `apt install gnupg2 pass` to `praqma/vscode`
- Add `rm /usr/bin/docker-credential-secretservice`to `praqma/vscode`
- Do one of these as step 1 of the exercise (feels wrong)

Anybody have any input on what solution is best?
